servicemanager/servicemanager.go

```diff
@@ -285,7 +303,32 @@ func (sm *ServiceManager) ValidateService(service ServiceInfo) error {
  * Private
  **********************************************************************************************************************/
 
+func (sm *ServiceManager) doRestoreService(serviceID string) error {
+	log.WithFields(log.Fields{"serviceID": serviceID}).Debug("Restore service")
+
+	service, err := sm.serviceInfoProvider.GetService(serviceID)
+	if err != nil {
+		return aoserrors.Wrap(err)
+	}
+
+	if !service.Cached {
+		log.Warn("Service was not cached")
```

Comment:
Use same format for layers and services:
```go
log.Warningf("layer %v not cached", digest)
```
or any other variant but same in both cases.

Reply:
Done